proposals/0209-collection-search.md

```diff
@@ -0,0 +1,66 @@
+# Collection search
```

Comment:
The name of the proposal doesn't cover its actual scope, which includes replacing.  Maybe "Collection search & replace"?

Reply:
Might as well throw "remove in there too "Collection search, remove & replace"

thanks!